docs/painless/painless-contexts/painless-runtime-fields-context.asciidoc

```diff
@@ -16,6 +24,9 @@ about how to use runtime fields.
         Contains the fields of the specified document where each field is a
         `List` of values.
 
+`emit` (`String`, read-only)::
```

Comment:
`emit` isn't a variable - its a method kind of like `grok` and `dissect`.

Reply:
Aha! I'll move `emit` to the *Methods* section.